bin/inx-chronicle/src/api/stardust/analytics/routes.rs

```diff
@@ -39,7 +39,6 @@ pub fn routes() -> Router {
                     "/blocks",
                     Router::new()
                         .route("/", get(block_analytics::<()>))
-                        .route("/milestone", get(block_analytics::<MilestonePayload>))
```

Comment:
Why do we remove this again?

Reply:
Well, it returns useless data pretty much.